### ISSUE TITLE
Project manager loads with the size settings of the previously opened project

### ISSUE THREAD

**Issue by Zylann** [godotengine/godot#4045]
Windows 7 64 bits, Godot 2.0.1

I quit "to project list", then the Project Manager opens in a tiny window that appears to have the same options as the last opened project.

My game had a tiny resolution and a viewport stretch mode. The project manager opened tiny too, and when resized it zooms without filter while keeping the ratio with black bands. Just as my game :p

Open this project: http://zylannprods.fr/dl/godot/ProjectManagerWrongDisplay.zip
Then "quit to project list", notice the weird window size and stretch behaviour.

**Comment by neikeq** [comment]
Can confirm the same bug on current HEAD with Ubuntu 15.10 64bits. Clicking _Quit to Project List_ will open the project manager with a window size equivalent to the project display settings (test_width and test_height, or width and height if the previous two are not set).

**Comment by mefihl** [comment]
Confirmed on Windows 10 64 bit

**Comment by manokara** [comment]
Confirmed on Arch Linux 64-bit, Godot 2.0.3.

**Comment by akien-mga** [comment]
That must be a regression, I don't remember it from before 2.0.

**Comment by akien-mga** [comment]
It's indeed a regression between 2.0 and 2.0.1, I'm currently bisecting the faulty commit and will try to fix it.

**Comment by akien-mga** [comment]
```
$ git bisect log
git bisect start
# bad: [53aa9cd51ab2551d67a71729ff61ae6668abda74] Release 2.0.1
git bisect bad 53aa9cd51ab2551d67a71729ff61ae6668abda74
# good: [9e2b9f8cb1c34fe7c3aa91bdd7f40ebb6a1a318f] bump to stable, congratulations everyone
git bisect good 9e2b9f8cb1c34fe7c3aa91bdd7f40ebb6a1a318f
# bad: [9ed3d21d5a3f0c5bad70b9a24d0fd2be8b6e81e8] Sync classes
git bisect bad 9ed3d21d5a3f0c5bad70b9a24d0fd2be8b6e81e8
# bad: [ee5c250b632c9067c6be6d121b50d62e83e0d3b0] OUYA gamepad mappings fix
git bisect bad ee5c250b632c9067c6be6d121b50d62e83e0d3b0
# good: [61f17fb1bb507a15a325ee4c23a69d85f9d8b7b6] applied some typo fixes. see godotengine/godot-docs#18
git bisect good 61f17fb1bb507a15a325ee4c23a69d85f9d8b7b6
# good: [d86b12a3973a46a0b6890276e40f3ff464baa5a2] Fixed closing a scene tab when it was not the CURRENT tab. Fixes #3810
git bisect good d86b12a3973a46a0b6890276e40f3ff464baa5a2
# bad: [e4d367e7a1fa6e8cefb95b57fe776cba13e84e75] theora on server build
git bisect bad e4d367e7a1fa6e8cefb95b57fe776cba13e84e75
# bad: [45a0bbe56e64ad14f18c037bf7130f3d383ec6d6] adds -pm and -project_manager command line options to start project manager
git bisect bad 45a0bbe56e64ad14f18c037bf7130f3d383ec6d6
# first bad commit: [45a0bbe56e64ad14f18c037bf7130f3d383ec6d6] adds -pm and -project_manager command line options to start project manager
```
```
45a0bbe56e64ad14f18c037bf7130f3d383ec6d6 is the first bad commit
commit 45a0bbe56e64ad14f18c037bf7130f3d383ec6d6
Author: Ariel Manzur <ariel@okamstudio.com>
Date:   Thu Feb 25 03:02:09 2016 -0300

    adds -pm and -project_manager command line options to start project manager
    
    fixes bug where the user has an engine.cfg on the executable directory so it runs the game instead of opening the project manager

:040000 040000 6282b3a45d1e3f87d954b79618dfa18af17aaac2 e095cfe74d9ac51cc2d20c361421b0618fbbdebf M      main
:040000 040000 c20d27f3d6a7f12aa29c0586288df42b10a8cdd7 de5302994198a242a5dd1132b5fd0682c1e752f1 M      tools
```
